ConsoleApp/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿Console.WriteLine("Compare method:");
+string str_1 = "aufwbferfu";
+string str_2 = "aufwbferfu";
+Console.WriteLine(Compare(str_1, str_2));
+Console.WriteLine();
+
+static bool Compare(string comp_1, string comp_2)
```

Comment:
name violation here, it should be
```suggestion
static bool Compare(string comp1, string comp2)
```